docs/Performance.md

```diff
@@ -73,6 +73,17 @@ This is unavoidable: a lot more work needs to be done at runtime to provide you
 When running a bundled app, these statements can cause a big bottleneck in the JavaScript thread.
 This includes calls from debugging libraries such as [redux-logger](https://github.com/evgenyrodionov/redux-logger),
 so make sure to remove them before bundling.
+You can also use this [babel plugin](https://babeljs.io/docs/plugins/transform-remove-console/) that removes all the `console.*` calls. You need to install it first using `yarn add babel-plugin-transform-remove-console`, and then edit the `.babelrc` file under your project directory like this:
```

Comment:
Can you use npm instead? We're all big fans of Yarn but we haven't yet updated our docs with Yarn instructions, so it's best if we use npm for consistency.

People who've installed Yarn are more likely to be aware of what the equivalent Yarn command is.